module1/homework/make_unique/make_unique-ut.cpp

```diff
@@ -0,0 +1,67 @@
+#include <gtest/gtest.h>
+#include <memory>
+#include "make_unique.hpp"
+
+constexpr int testIntValue = 10;
+constexpr double testDoubleValue = 3.14;
+constexpr bool testBoolValue = true;
+constexpr size_t arraySize = 5;
+
+template<class Type> struct default_delete;
+template<class Type> struct default_delete<Type[]>;
+
+struct make_uniqueTest {
+public:
+    make_uniqueTest();
+    make_uniqueTest(int intValue, double doubleValue, bool boolValue)
+    : intValue_(intValue), doubleValue_(doubleValue), boolValue_(boolValue)
+    {}
+
+    bool operator==(const make_uniqueTest& other) const {
+        return intValue_ == other.intValue_
+            && doubleValue_ == other.doubleValue_
+            && boolValue_ == other.boolValue_;
+    }
+
+private:
+    int intValue_;
+    double doubleValue_;
+    bool boolValue_;
```

Comment:
initialize values to avoid undefined behavior when using default constructor.

Reply:
done